doc/nrf/device_guides/wifi_coex.rst

```diff
@@ -242,6 +242,72 @@ To enable the generic three-wire coexistence, do the following:
    * :kconfig:option:`CONFIG_MPSL_CX`
    * :kconfig:option:`CONFIG_MPSL_CX_3WIRE`
 
+.. _ug_radio_mpsl_cx_generic_1wire:
+
+Generic one wire coexistence
+============================
+
+Refer to :ref:`ug_radio_coex_mpsl_cx_based` for the general requirements of this implementation.
+
+The generic one wire coexistence interface can be used to allow the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` to coexist alongside an LTE device on a separate chip.
+It is specifically designed for the coex interface of the nRF91 Series SiP.
+
+Hardware description
+--------------------
+
+The generic one wire interface consists of the signals listed in the table below.
+The *Pin* is a generic pin name of a PTA, identified rather by its function.
+The *Direction* is from the point of view of the SoC running the coexistence protocol.
+The *DT property* is the name of the devicetree node property that configures the connection between the SoC running the coexistence protocol and the other device.
+
+.. table:: Generic one wire coexistence protocol pins
+
+   ============  =========  =================================  ==============
+   Pin           Direction  Description                        DT property
+   ============  =========  =================================  ==============
+   GRANT         In         Grant signal                       grant-gpios
+   ============  =========  =================================  ==============
+
+Enabling generic one wire coexistence
+-------------------------------------
+
+To enable the generic one wire coexistence, do the following:
+
+
+1. Add the following node to the devicetree source file:
+
+   .. code-block::
+
+      / {
+            nrf_radio_coex: radio_coex_one_wire {
+               status = "okay";
+               compatible = "generic-radio-coex-one-wire";
+               grant-gpios =   <&gpio0 25 GPIO_ACTIVE_LOW>;
+         };
+      };
+
+#. Optionally, if not using the nRF91 Series SiP, the ``GRANT`` signal may be configured active-high using ``GPIO_ACTIVE_HIGH``
+#. Optionally, replace the node name ``radio_coex_one_wire`` with a custom one.
+#. Replace the pin numbers provided for each of the required properties:
+
+   * ``grant-gpios`` - GPIO characteristic of the device that controls the ``GRANT`` signal of the PTA (RF medium access granted).
```

Comment:
```suggestion
#. Replace the pin number provided for the ``grant-gpios`` property.
   This is the GPIO characteristic of the device that controls the ``GRANT`` signal of the PTA (RF medium access granted).
```
Since there's only one property to configure.

Reply:
also replaced "controls" with "interfaces with" because it's an input GPIO